docs/command_reference.md

```diff
@@ -0,0 +1,299 @@
+    - **Targets**: The targets role's signature indicates which target files are trusted by clients.
+    - **Delegations**: the targets role can delegate full or partial trust to other roles.  Delegating trust means that the targets role indicates another role (that is, another set of keys and the threshold required for trust) is trusted to sign target file metadata.
+    - **Snapshot**:  The snapshot role signs a metadata file that provides information about the latest version of all of the other metadata on the trusted collection (excluding the timestamp file, discussed below).
+    - **Timestamp**:  To prevent an adversary from replaying an out-of-date signed metadata file whose signature has not yet expired, an automated process periodically signs a timestamped statement containing the hash of the snapshot file.
+
+To read further about the framework Notary implements, check out [The Update Framework](https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.txt)
+
+## Command Reference
+
+### Set up Notary CLI
+
+Once you install the Notary CLI client, you can use it to manage your signing
+keys, authorize other team members to sign content, and rotate the keys if
+a private key has been compromised.
+
+When using the Notary CLI client you need to specify where the URL of the Notary server
+you want to communicate is with the `-s` flag, and where to store the private keys and cache for
+the CLI client with the `-d` flag.  There are also fields in the [client configuration](/reference/client-config.md) to set these variables.
+
+```bash
+# Create an alias to always have the notary client talking to the right notary server
+$ alias notary="notary -s <notary_server_url> -d <notary_cache_directory>
+```
+
+When working Docker Content Trust, it is important to specify notary's client cache as `~/.docker/trust`.  Also, Docker Hub provides its own Notary server located at `https://notary.docker.io`, which contains trust data for many images including official images, though you are welcome to use your own notary server.
+
+## Initializing a trusted collection
+
+Notary can initialize a trusted collection with the `notary init` command:
+```bash
+$ notary init <GUN>
+```
+
+This command will generate targets and snapshot keys locally for the trusted collection, and try to locate a root key to use in the specified notary client cache (from `-d` or the config).
+If notary cannot find a root key, it will generate one.  For all keys, notary will also prompt for a passphrase to encrypt the private key material at rest.
+
+If you'd like to initialize your trusted collection with a specific root key, there is a flag to provide it.  Notary will require that the key is encrypted:
+```bash
+$ notary init <GUN> --rootkey <key_file>
+```
+
+Note that you will have to run a publish after this command for it to take effect, as there are staged changes to initialize the trusted collection that have not been pushed to a notary server.
```

Comment:
nitpick:  "as it creates staged changes..." might be clearer about where those staged changes came from.